### PR TITLE
Daniel/series scroll animation

### PR DESCRIPTION
**Pull request by dantheli**
Adds scrolling animation to series detail header. I was on the bus and had some free time.
scroll animation

### CODE REVIEW

File: Podcast/SeriesDetailHeaderView.swift [hunk 1]
```diff
@@ -178,13 +193,15 @@ class SeriesDetailHeaderView: UIView, UICollectionViewDelegate, UICollectionView
...
+            make.bottom.equalToSuperview().inset(tagsViewTopOffset)
         }
         
         episodeSeparator.snp.makeConstraints { make in
-            make.centerX.equalToSuperview()
             make.height.equalTo(episodeSeparatorHeight)
-            make.width.equalToSuperview()
-            make.top.equalTo(tagsCollectionView.snp.bottom).offset(tagsViewTopOffset)
+            make.top.equalTo(contentContainer.snp.bottom)
+            make.leading.equalToSuperview()
```
Comment (natashaarmbrust) [inline Podcast/SeriesDetailHeaderView.swift]: `make.leading.trailing.bottom.equalToSuperview()` instead of 3 lines

**Comment by doctor3w** [comment]
This is sick

**Review by doctor3w — APPROVED** [review]

**Review by natashaarmbrust — COMMENTED** [review]
So dope Dan!

File: Podcast/SeriesDetailHeaderView.swift (hunk 1 above)
Comment (dantheli) [reply Podcast/SeriesDetailHeaderView.swift]: holy...

**Pull request merged**